### PR TITLE
Double solenoid storage

### PR DESCRIPTION
**Pull request by yoshiyahoo**

### CODE REVIEW

**Review by MichaelCrandall — DISMISSED** [review]
You commited a change to the .xml file, please remove this commit from your pr

**Comment by yoshiyahoo** [comment]
I removed the file from the commit

File: src/main/java/frc/robot/subsystems/Storage.java [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
+
 import edu.wpi.first.wpilibj.DigitalInput;
 
 public class Storage extends SubsystemBase {
 
-    public static final DigitalInput ballsInSwitch = new DigitalInput(StorageConstants.BALLS_IN_SWITCH_ID);
-    public static final WPI_VictorSPX polycord = new WPI_VictorSPX(StorageCosntants.POLYCORD_MOTOR_ID);
+    private static final DigitalInput ballsInSwitch = new DigitalInput(StorageConstants.BALLS_IN_SWITCH_ID);
+    private static final WPI_VictorSPX polycord = new WPI_VictorSPX(StorageConstants.POLYCORD_MOTOR_ID);
+    private DoubleSolenoid m_doubleSolenoid = new DoubleSolenoid(3,0,1);
```
Comment (MichaelCrandall) [inline src/main/java/frc/robot/subsystems/Storage.java]: all of these should be constants

**Review by MichaelCrandall — CHANGES_REQUESTED** [review]
Master was updated so you need to pull from master to get the most recent code

**Review by MichaelCrandall — APPROVED** [review]

**Pull request merged**